osbs/conf.py

```diff
@@ -491,3 +491,13 @@ def get_can_orchestrate(self):
     def get_info_url_format(self):
         return self._get_value("info_url_format", self.conf_section,
                                "info_url_format")
+
+    def get_low_priority_node_selector(self):
+        nodeselector = {}
+        nodeselector_str = self._get_value("low_priority_node_selector", self.conf_section,
+                                           "low_priority_node_selector")
+        if nodeselector_str:
+            nodeselector = dict([[x.strip() for x in constraint.split('=', 1)]
+                                 for constraint in nodeselector_str.split(',')])
```

Comment:
My preference would be to keep to only one `for` in each comprehension, but otherwise this looks fine.